tests/run.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+
+import dj_database_url
+import django
+from colour_runner.django_runner import ColourRunnerMixin
+from django.conf import settings
+from django.test.runner import DiscoverRunner
+
+
+BASEDIR = os.path.dirname(os.path.dirname(__file__))
+
+settings.configure(
+    DATABASES={
+        'default': dj_database_url.config(
+            default='sqlite://{}/user_deletion.db'.format(BASEDIR),
```

Comment:
No reason, as I just need to notify and delete users, I thought it was enough, what do you think?